Wikipedia/Code/WMFAppViewController.m

```diff
@@ -1938,6 +1939,7 @@ - (nonnull WMFSettingsViewController *)settingsViewController {
         WMFSettingsViewController *settingsVC =
             [WMFSettingsViewController settingsViewControllerWithDataStore:self.dataStore];
         [settingsVC applyTheme:self.theme];
+        _settingsViewController.notificationsCenterPresentationDelegate = self;
```

Comment:
Also haven't looked into this, but when I turn off the Explore feed and tap the notifications button on the Settings screen, it doesn't navigate away. So it seems like this delegate might be nil in that case.

Reply:
This was just a silly ordering mistake by me – updated.